src/ec/packages/ec-component-social-media-share/social-media-share.story.js

```diff
@@ -9,26 +15,125 @@ import demoData from './demo/data';
 import SocialMediaShare from './ecl-social-media-share.html.twig';
 import notes from './README.md';
 
-demoData.links.forEach(link => {
-  if (link.icon) {
-    link.icon.forEach(icon => {
-      icon.path = defaultSprite; // eslint-disable-line no-param-reassign
-    });
-  }
-});
+const prepareSocialMediaShare = data => {
+  data.description = text(
+    'description',
+    data.description,
+    buttonLabels.required
+  );
+
+  data.links.forEach((link, i) => {
+    let label = buttonLabels.optional;
+    // Firt item we make it mandatory and not removable.
+    if (i === 0) {
+      label = buttonLabels.required;
+      link.label = select(
```

Comment:
Because you can only choose between a defined set of social networks we provide the icons for, and you can't decide how to label them, basically, at least as ECL, than if you add your own you do it, but this is ecl twig :)
The thing is that this is the same i did also for the social network follow and you merged it without asking me, but this is the reason :)

Reply:
ok make sense 👍